CHANGELOG.md

```diff
@@ -7,10 +7,7 @@
 - 162c4e8: Spinner swap aria label for alt
 - 0b2a12f: add theming of navbar (desktop)
 
-### Patch Changes
-
 - 737d830: Bump minor
-
 ## 5.53.0
 
 ### Minor Changes
```

Comment:
you also need to generate and push changeset file (one with a weird name)

Reply:
how do you do that?

Reply:
use `pnpm changeset` https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md
it should generate a new file and changelog from above as well